package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "appium-inspector",
-  "version": "2022.2.1",
+  "version": "2022.2.2",
```

Comment:
Could you exclude this change?

Reply:
this is done in the latest commit, thank you .